03-rnaseq/ortholog-mapping_rnaseq_01_ensembl.Rmd

```diff
@@ -187,38 +187,51 @@ The [HGNC database](https://www.genenames.org/) currently contains over 39,000 p
 
 The [HGNC Comparison of Orthology Predictions (HCOP)](https://www.genenames.org/tools/hcop/) is a search tool that combines orthology predictions for a specified human gene, or set of human genes from a variety of sources, including Ensembl Compara, HGNC, and NCBI Gene Orthology [@Wright2005].
 In general, an orthology prediction where most of the databases concur would be considered the reliable, and we will use this to prioritize mapping in cases where there is more than one possible ortholog for a gene.
-HCOP was originally designed to show orthology predictions between human and mouse, but has been expanded to include data from 18 genomes [@hcop-help].
+HCOP was originally designed to show orthology predictions between human and mouse, but has been expanded to include data from 18 genomes, including mouse, which we will use in this notebook [@hcop-help].
 
-First, we need to download the file from the server holding the HGNC data.
-Go to this [directory page of the HGNC Comparison of Orthology Predictions (HCOP) files](ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/).
+We can automatically download the human mouse file we need for this example using `download.file()` command. 
```

Comment:
Oh, I meant to make this comment in the other one but it slipped my mind... it isn't really automatic. (port back to the other file as well, please?)

```suggestion
We can download the human mouse file we need for this example using `download.file()` command. 
```

Reply:
Yes, I will add this change to the other file too.